### DISCUSSION TITLE
File saving dialogue: Ability to return to original file name.

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#254] [Ideas]
After accidentally pressing a button while the file name of the "save as" dialogue is selected, there is no way to go back to the originally provided file name without tediously re-opining the dialogue. CTRL+Z does not work on the file saving dialogue.

There needs to be a button to return to the original file name, or CTRL-Z should work there.

**Comment by JosephMcc** [comment]
The file save dialog is provided by whatever toolkit is used to build an application. Linux Mint doesn't have control over this.
